src/jit/bitset.cpp

```diff
@@ -12,10 +12,12 @@
 #include "bitsetasshortlong.h"
 #include "bitsetasuint64inclass.h"
 
+// clang-format off
```

Comment:
We have some tables that we would like to not be reformatted, so they are surrounded in clang-format off/clang-format on